### PR TITLE
[Doc] Add Generative AI tag to examples gallery

### LINKED ISSUE #38076: [Example Gallery] - Add a generative AI filter in the left nav
### Description

As per @pcmoritz's request,  we plan to introduce an additional filter in the example gallery, specifically dedicated to Generative AI. Presently, the LLM filter includes both LLM and image generation/generative AI examples, but we aim to categorize them separately. Below is a list of examples along with the corresponding filters they should belong to. 

https://www.anyscale.com/blog/ray-common-production-challenges-for-generative-ai-infrastructure -  LLM, Gen AI 

https://www.anyscale.com/blog/training-175b-parameter-language-models-at-1000-gpu-scale-with-alpa-and-ray - LLM, GEN AI

https://www.anyscale.com/blog/faster-stable-diffusion-fine-tuning-with-ray-air - GEN AI

https://www.anyscale.com/blog/how-to-fine-tune-and-serve-llms - LLM, GEN AI

https://www.businessinsider.com/openai-chatgpt-trained-on-anyscale-ray-generative-lifelike-ai-models-2022-12 - LLM, GEN AI

https://docs.ray.io/en/latest/ray-air/examples/gptj_deepspeed_fine_tuning.html - LLM, GEN AI 

https://github.com/ray-project/aviary/ - LLM, GEN AI 

https://docs.ray.io/en/latest/ray-air/examples/dreambooth_finetuning.html - GEN AI 

https://docs.ray.io/en/latest/ray-air/examples/stablediffusion_batch_prediction.html - GEN AI 

https://docs.ray.io/en/latest/ray-air/examples/gptj_serving.html - LLM and GEN AI 

https://www.anyscale.com/blog/turbocharge-langchain-now-guide-to-20x-faster-embedding - LLM and GEN AI 

https://docs.ray.io/en/latest/ray-air/examples/dolly_lightning_fsdp_finetuning.html#dolly-lightning-fsdp-finetuning - LLM and GEN AI

Also,[ a link to the design for the new Generative AI filter](https://www.figma.com/file/fRYp3ZBGbpg9dlu3WlBYY9/Quansight---Example-gallery-and-CSAT?type=design&node-id=1%3A145&mode=design&t=4T9E3iU679FNAGq2-1)

### Link

_No response_

### PR DESCRIPTION
**Pull request by peytondmurray**
## Why are these changes needed?

This PR adds a new tag to the example gallery: `Generative AI`. Some duplicate gallery cards are also removed, and a broken link is fixed.

## Related issue number

Closes https://github.com/ray-project/ray/issues/38076.

## Checks

- [x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [x] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [x] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [x] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by simran-2797** [comment]
https://anyscale-ray--38099.com.readthedocs.build/en/38099/ray-air/examples/stablediffusion_batch_prediction.html - Should be GEN AI instead of LLM as per the list mentioned in the issue. 

Can we add an GEN AI tag to these examples - https://anyscale-ray--38099.com.readthedocs.build/en/38099/train/examples/lightning/vicuna_13b_lightning_deepspeed_finetune.html

**Comment by peytondmurray** [comment]
> https://anyscale-ray--38099.com.readthedocs.build/en/38099/ray-air/examples/stablediffusion_batch_prediction.html - Should be GEN AI instead of LLM as per the list mentioned in the issue.

Ahh, looks like you found a duplicate. The other one was already updated.

Updated the deepspeed example too.

**Review by simran-2797 — APPROVED** [review]

File: doc/source/ray-overview/examples.rst [hunk 1]
```diff
@@ -136,13 +136,6 @@ Ray Examples
 
         Perform batch forecasting on NYC Taxi Dataset with Prophet, ARIMA and Ray AIR
 
-    .. grid-item-card:: :bdg-secondary:`Code example`
-        :class-item: gallery-item
-        :link: /ray-air/examples/gptj_deepspeed_fine_tuning
```
Comment (pcmoritz) [inline doc/source/ray-overview/examples.rst:141]: @richardliaw Can you have a look if we actually intended to remove this? If no we should bring it back with the updated link :)

File: doc/source/ray-overview/examples.rst [hunk 2]
```diff
@@ -311,27 +304,6 @@ Ray Examples
 
         Serving an Object Detection Model
 
-    .. grid-item-card:: :bdg-secondary:`Code example`
```
Comment (pcmoritz) [inline doc/source/ray-overview/examples.rst:314]: @richardliaw We probably didn't intend to remove this, did we?

File: doc/source/ray-overview/examples.rst [hunk 3]
```diff
@@ -311,27 +304,6 @@ Ray Examples
...
-        :class-item: gallery-item llm
-        :link: /ray-air/examples/dreambooth_finetuning
-        :link-type: doc
-
-        Fine-tuning DreamBooth with Ray AIR
-
-    .. grid-item-card:: :bdg-secondary:`Code example`
```
Comment (pcmoritz) [inline doc/source/ray-overview/examples.rst:321]: Same for this

File: doc/source/ray-overview/examples.rst [hunk 4]
```diff
@@ -311,27 +304,6 @@ Ray Examples
...
-        :class-item: gallery-item llm
-        :link: /ray-air/examples/stablediffusion_batch_prediction
-        :link-type: doc
-
-        Stable Diffusion Batch Prediction with Ray AIR
-
-    .. grid-item-card:: :bdg-secondary:`Code example`
```
Comment (pcmoritz) [inline doc/source/ray-overview/examples.rst:328]: What about this?

File: doc/source/ray-overview/examples.rst (hunk 2 above)
Comment (peytondmurray) [reply doc/source/ray-overview/examples.rst:314]: Nope, this was intentional, it's a duplicate: https://github.com/peytondmurray/ray/blob/918de4f5a15d3bfd7aaaf7c7ff4d83d067a1806f/doc/source/ray-overview/examples.rst?plain=1#L153

File: doc/source/ray-overview/examples.rst (hunk 3 above)
Comment (peytondmurray) [reply doc/source/ray-overview/examples.rst:321]: This is also a duplicate: https://github.com/peytondmurray/ray/blob/918de4f5a15d3bfd7aaaf7c7ff4d83d067a1806f/doc/source/ray-overview/examples.rst?plain=1#L188

File: doc/source/ray-overview/examples.rst (hunk 4 above)
Comment (peytondmurray) [reply doc/source/ray-overview/examples.rst:328]: This is another duplicate: https://github.com/peytondmurray/ray/blob/918de4f5a15d3bfd7aaaf7c7ff4d83d067a1806f/doc/source/ray-overview/examples.rst?plain=1#L146

File: doc/source/ray-overview/examples.rst (hunk 1 above)
Comment (peytondmurray) [reply doc/source/ray-overview/examples.rst:141]: This is intended, there was a duplicate: https://github.com/peytondmurray/ray/blob/918de4f5a15d3bfd7aaaf7c7ff4d83d067a1806f/doc/source/ray-overview/examples.rst?plain=1#L63.

**Review by pcmoritz — APPROVED** [review]

**Pull request merged**